cmake/GenerateHelptags.cmake.in

```diff
@@ -1,10 +1,10 @@
 if(DEFINED ENV{DESTDIR})
   file(TO_CMAKE_PATH
-    "$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    "$ENV{DESTDIR}/@CMAKE_INSTALL_FULL_DATADIR@/nvim/runtime/doc"
```

Comment:
This is how cmake upstream recommend I go about doing this. I don't disagree there may be others; but I asked them if that was possible and they said no, so maybe they just didn't want me to do something kludgy.